classes/user/Collector.php

```diff
@@ -246,6 +248,17 @@ public function includeReviewerData(bool $includeReviewerData = true): self
         return $this;
     }
 
+    /**
+     * Defines whether active submissions count should be included
+     */
+    public function includeActiveSubmissions(int $userGroupId): self
```

Comment:
What about naming this `includeActiveSubmissionsByUserGroupIds`? And passing the array of IDs instead of int?

Reply:
But that's on your opinion

Reply:
I have renamed the Function name, however the I kept the param as int as it is going to be always one group id when filtering the users?